apps/key-manager/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@apps/key-manager",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "build": "cloudflare",
+    "cf-typegen": "wrangler types --env-interface CloudflareEnv env.d.ts",
```

Comment:
Might be best for this to be a `postinstall` script, or have the `postinstall` script call this.